package/yast2-packager.spec

```diff
@@ -87,18 +87,9 @@ Requires:       yast2-ruby-bindings >= 1.0.0
 Summary:	YaST2 - Package Library
 
 
```

Comment:
It's [here](https://github.com/yast/yast-packager/pull/161/commits/3b40ea7e6ee76a67d8cf5856ad6cc097612a032d).